### PR TITLE
Troposphere and sutherland model updates

### PR DESCRIPTION
**Pull request by pgkirsch**
Didn't mean to put these on the same branch.

### CODE REVIEW

File: sutherland.py [hunk 1]
```diff
@@ -0,0 +1,33 @@
...
+        T  = Variable("T", "K", "Temperature")
+
+        # Constants
+        T_S   = Variable("T_S", TS, "K", "Sutherland Temperature")
+        C_1 = Variable("C_1", C1, "kg/(m*s*K^0.5)",
+                       "Sutherland coefficient")
+
+        objective = mu
+        constraints = [((T+T_S).mono_approximation({T:288.15,
+                                                    T_S:T_S.value.magnitude}))
```
Comment (whoburg) [inline sutherland.py]: hmm, this is cool, but it makes the approximation around `t=288.15`. Is there reason to believe that's a valid approximation? Over what range of `T` is the approximation good?

File: troposphere.py [hunk 2]
```diff
@@ -6,10 +6,13 @@ class Troposphere(Model):
     """
     Density as a function of altitude based on standard atmosphere model
 
-    Required pressures: upward pressure on rho and h
-    Assumptions: only valid to the top of the troposphere (~20km)
+    Required pressures: downward pressure on rho
+    Assumptions: only valid to the top of the troposphere (~11km)
```
Comment (whoburg) [inline troposphere.py]: I think the "Required pressures" given here are much clearer than the `Assumptions` section of the current Sutherland docstring.

File: sutherland.py (hunk 1 above)
Comment (pgkirsch) [inline sutherland.py]: Not particularly rigorous, but I did plot it to make sure it looks ok over a reasonable range of temperatures (200-300K). The offset is just due to rounding.
screen shot 2016-03-04 at 01 46 26 am

Comment (whoburg) [inline sutherland.py]: pretty convincing! I think that needs to make its way into the documentation of the model -- e.g. list the bounds on `T` over which the error is less than 1%, less than 5%, etc.

@bqpd, here's an interesting model documentation use case -- plots

Comment (bqpd) [inline sutherland.py]: Supported (using markdown image syntax) in #547

**Pull request closed without merging**